{
  "Header": {
    "Original": "CVE-2020-1752 (A use-after-free vulnerability introduced in glibc upstream version 2. ...)",
    "Line": 211585,
    "ID": "CVE-2020-1752",
    "Description": "(A use-after-free vulnerability introduced in glibc upstream version 2. ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-3152-1}",
      "Line": 211586,
      "Type": "xref",
      "Bugs": [
        "DLA-3152-1"
      ]
    },
    {
      "Original": "- glibc 2.30-3 (bug #953788)",
      "Line": 211587,
      "Type": "package",
      "Package": "glibc",
      "Kind": "fixed",
      "Version": "2.30-3",
      "BugNo": 953788
    },
    {
      "Original": "[stretch] - glibc \u003cno-dsa\u003e (Minor issue)",
      "Line": 211588,
      "Type": "package",
      "Release": "stretch",
      "Package": "glibc",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "[jessie] - glibc \u003cno-dsa\u003e (Minor issue)",
      "Line": 211589,
      "Type": "package",
      "Release": "jessie",
      "Package": "glibc",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://sourceware.org/bugzilla/show_bug.cgi?id=25414",
      "Line": 211590,
      "Type": "NOTE",
      "Description": "https://sourceware.org/bugzilla/show_bug.cgi?id=25414"
    },
    {
      "Original": "NOTE: Introduced in: https://sourceware.org/git/gitweb.cgi?p=glibc.git;h=f2962a71959fd254a7a223437ca4b63b9e81130c (2.14)",
      "Line": 211591,
      "Type": "NOTE",
      "Description": "Introduced in: https://sourceware.org/git/gitweb.cgi?p=glibc.git;h=f2962a71959fd254a7a223437ca4b63b9e81130c (2.14)"
    },
    {
      "Original": "NOTE: Fixed by: https://sourceware.org/git/gitweb.cgi?p=glibc.git;h=ddc650e9b3dc916eab417ce9f79e67337b05035c",
      "Line": 211592,
      "Type": "NOTE",
      "Description": "Fixed by: https://sourceware.org/git/gitweb.cgi?p=glibc.git;h=ddc650e9b3dc916eab417ce9f79e67337b05035c"
    }
  ]
}